pytorch_lightning/trainer/evaluation_loop.py

```diff
@@ -388,6 +395,9 @@ def _evaluate(
         # log callback metrics
         self.__update_callback_metrics(eval_results, using_eval_result)
 
+        # Write predictions to disk if they're available.
+        predictions.to_disk()
```

Comment:
```suggestion
        predictions.flush()
```
maybe a better name...